lib/string_reverse.rb

```diff
@@ -20,3 +20,12 @@ def string_reverse(my_string)
 
   my_string << "not implemented"
 end

```

Comment:
Minor: Since this function is reversing the string in-place, you do not need to return `my_string`.